### PR TITLE
Added index.php

### PR DESCRIPTION
**Pull request by andymelichar**
Initial commit of index.php

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?PHP
+
+echo "Hi there, here is some code!";
+
+$apples = 12;
+$oranges = 23;
+
+$totalFruit = $apples + $oranges + $grapes;
```
Comment (andymelichar) [inline index.php]: Where are the grapes?

Comment (a2hostingwebdev) [inline index.php]: $grapes is not defined anywhere, so this calculation is incorrect.

**Review by a2hostingwebdev — CHANGES_REQUESTED** [review]
I don't think you should merge this until you fix the issue with the grapes.

File: index.php (hunk 1 above)
Comment (andymelichar) [reply index.php]: Thanks! I added some grapes so this should be fixed now.

**Review by andymelichar — COMMENTED** [review]
Added grapes.

**Comment by a2hostingwebdev** [comment]
Looks good to me! Go ahead with your commit!

**Pull request merged**